From f10a5ab0b3a4cb8a7e17a4935fb33d283c3dd31f Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@sandeen.net>
Date: Tue, 20 Sep 2016 08:48:54 +1000
Subject: [PATCH] mkfs.xfs: clarify ftype defaults in manpage

When CRCs were made default, a few leftovers related to its
prior non-default status remained in the manpage, in the ftype
section.  Clean those up, stating the correct default for this
feature.

Reported-by: Chris Murphy <chris@cmurf.com>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
Reviewed-by: Dave Chinner <dchinner@redhat.com>
Signed-off-by: Dave Chinner <david@fromorbit.com>
---
 man/man8/mkfs.xfs.8 | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

Index: xfsprogs-rhel7.5/man/man8/mkfs.xfs.8
===================================================================
--- xfsprogs-rhel7.5.orig/man/man8/mkfs.xfs.8
+++ xfsprogs-rhel7.5/man/man8/mkfs.xfs.8
@@ -587,13 +587,11 @@ do not need to look up the inode to dete
 
 The
 .I value
-is either 0 or 1, with 1 signifiying that filetype information
-will be stored in the directory structure. The default value is 0.
+is either 0 or 1, with 1 signifying that filetype information
+will be stored in the directory structure.  The default value is 1.
 
-When CRCs are enabled via
-.B \-m crc=1,
-the ftype functionality is always enabled. This feature can not be turned
-off for such filesystem configurations.
+When CRCs are enabled (the default), the ftype functionality is always
+enabled, and cannot be turned off.
 .IP
 .RE
 .TP
